app/src/organisms/PipetteWizardFlows/Carriage.tsx

```diff
@@ -1,24 +1,68 @@
 import * as React from 'react'
 import { Trans, useTranslation } from 'react-i18next'
 import capitalize from 'lodash/capitalize'
-import { SPACING } from '@opentrons/components'
+import {
+  COLORS,
+  SPACING,
+  TEXT_TRANSFORM_CAPITALIZE,
+} from '@opentrons/components'
 import { SINGLE_MOUNT_PIPETTES } from '@opentrons/shared-data'
 import { StyledText } from '../../atoms/text'
 import { GenericWizardTile } from '../../molecules/GenericWizardTile'
+import { SimpleWizardBody } from '../../molecules/SimpleWizardBody'
+import { PrimaryButton, SecondaryButton } from '../../atoms/buttons'
 import unscrewCarriage from '../../assets/images/change-pip/unscrew-carriage.png'
 import { FLOWS } from './constants'
+import { CheckZAxisButton } from './CheckZaxisButton'
 
-import type { PipetteWizardStepProps } from './types'
+import type { PipetteWizardStepProps, ZAxisScrewStatus } from './types'
 
 export const Carriage = (props: PipetteWizardStepProps): JSX.Element | null => {
   const { goBack, proceed, flowType, selectedPipette } = props
   const { t } = useTranslation(['pipette_wizard_flows', 'shared'])
+  const [
+    zAxisScrewStatus,
+    setZAxisScrewStatus,
+  ] = React.useState<ZAxisScrewStatus>('unknown')
+  const [numberOfTryAgains, setNumberOfTryAgains] = React.useState<number>(0)
 
+  React.useEffect(() => {
+    if (zAxisScrewStatus === 'attached' || zAxisScrewStatus === 'detached')
+      proceed()
+  }, [proceed, zAxisScrewStatus])
+
+  const handleErrorTryAgain = (): void => {
+    setZAxisScrewStatus('attached')
```

Comment:
change this line to the following in order to test out the `try again` logic 

```suggestion
    setZAxisScrewStatus('stillAttached')
```